packages/core/src/button-bar/button-bar.ts

```diff
@@ -19,6 +19,7 @@ export type ButtonBarAlign = 'start' | 'center' | 'end' | 'space-between';
  */
 export class ButtonBar extends LitElement {
   /** @private */
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
You can remove this; there's a timing issue where if you haven't built `button-bar.scss.ts`, then it doesn't know the type of `styles`.

Reply:
right, I just forgot about it :)